src/services/services.ts

```diff
@@ -5807,9 +5908,11 @@ namespace ts {
             // Maps from a symbol ID to the ReferencedSymbol entry in 'result'.
             const symbolToIndex: number[] = [];
 
+            const indexToSymbol: {[index: number]: Symbol} = {};

```

Comment:
consider factoring this part out into a new function and calling this directly from getimplementations